apps/agentic/CLAUDE.md

```
@@ -59,6 +59,14 @@ Check configuration for errors and warnings without modifying.
 - Shows advisory warnings for deprecated keys, invalid values, etc.
 - Exit code 0 even with warnings (non-fatal)
 
+### `install [--path PATH] [--force]`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)